laravel/resources/views/emails/admin-removed-shift.blade.php

```diff
@@ -0,0 +1,9 @@
+<h1>Hi {{ $user_name }}!</h1>
+
+<p>
+    It seems you've been effected by a recent scheduling change for the {{ $shift_name }} shift.
```

Comment:
Affected* :sweat_smile: 

I think the wording and variables displayed could be made a bit more clear `There was a scheduling change and you were removed from the {{ $shift_name }} shift at {{ $shift_datetime }}.`